app/src/main/java/com/aileyzhang/musicapp/views/SongListItemView.java

```diff
@@ -0,0 +1,26 @@
+package com.aileyzhang.musicapp.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.aileyzhang.musicapp.R;
+
+/**
+ * Created by Ailey on 2017-12-18.
+ */
+
+public class SongListItemView extends View {
```

Comment:
Try extending a `ViewGroup`, such as `FrameLayout`. The view your custom view class extends is essentially the "parent" view that will contain your layout, since you usually inflate your XML layout then add it to "this" (`SongListItemView`) view.